drivers/pinctrl/cirrus/Kconfig

```diff
@@ -1,15 +1,4 @@
 # SPDX-License-Identifier: GPL-2.0-only
-config PINCTRL_CS42L43
-	tristate "Cirrus Logic CS42L43 Pinctrl Driver"
-	depends on MFD_CS42L43
-	select GPIOLIB
-	select PINMUX
-	select PINCONF
-	select GENERIC_PINCONF
-	help
-	  Select this to support the GPIO/Pinctrl functions of the Cirrus
-	  Logic CS42L43 PC CODEC.
-
```

Comment:
You probably need a different way of handling the changes that roll-back and re-add. This would be problematic for any git bisect with intermediate commits removing functionality.

Reply:
There is no intention of actually updating cs42l43 to use the SDCA stuff certainly in the short term and even if we did we would likely do so in a keeping both options available way. This is just to provide me a test platform.

Reply:
Worth saying I included the code in the chain in case you guys wanted to try things out, this should all work with the cs42l43's you have. Although you would need some more changes I haven't pushed yet to get audio out, but this will probe the device and setup the register map.